frontend/components/App/App.tsx

```diff
@@ -61,6 +63,8 @@ const App = ({ children, location }: IAppProps): JSX.Element => {
     setCurrentUser,
     setConfig,
     setEnrollSecret,
+    setABMExpiry,
+    setAPNsExpiry,
```

Comment:
We could also cleanup sandbox code while we're at it

Reply:
I didn't want to scope creep cleaning up the entire `<SandboxGate/>` in this banners PR estimated to be a 2 right before I took a week vacation.